kde-frameworks/kdelibs4support/kdelibs4support-5.109.0-r1.ebuild

```diff
@@ -13,12 +14,11 @@ DESCRIPTION="Framework easing the development transition from KDELibs 4 to KF 5"
 
 LICENSE="LGPL-2+"
 KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~x86"
-IUSE="X"
+IUSE="X network-status"
```

Comment:
- Keep `IUSE` sorted alphanumerically at all times
- Our global flag for `net-misc/networkmanager` is `networkmanager`, use that please.